CS483/CS483/Kartaclysm/Services/IO/InputActionMapping.cpp

```diff
@@ -109,59 +109,70 @@ namespace Kartaclysm
 			return;
 		}
 
-		HeatStroke::Event* pEvent = new HeatStroke::Event("PlayerInput_" + p_iPlayer);
+		// Helpful variables to avoid repetition
+		std::string strPlayerIdentifier = "P" + std::to_string(p_iPlayer) + "_";
```

Comment:
Why does the playerId have the "_" at the end? Surely it's not relevant to the id, and if the id ever needs to be concatenated with something else, such as "DriverX" for the abilities, it should be the responsibility of the concatenating code to insert the "_".